spec/requests/admin/impersonate_controller_spec.rb

```diff
@@ -106,4 +118,34 @@
       end
     end
   end
+
+  describe "#destroy" do
+    before { sign_in(admin) }
```

Comment:
We technically don't need this because the controller action bypasses the `ensure_admin` before action.